CONTRIBUTING.rst

```diff
@@ -69,6 +69,10 @@ Style guide
 
 These are the guidelines that we use for Aiven Docs; following them helps us keep all the content consistent and easy to follow for our audience. We review your pull request against these guidelines, so for fewer rounds of revisions, follow this advice!
 
+.. note::
```

Comment:
Would it be better to instead put this in the introduction? My thought is that the introduction would be revised to explain that the Google developer documentation style guide is the default and that exceptions/gaps are detailed in the rest of the style guide.

Reply:
Thanks @staceysalamon-aiven :-) It's an option definitely. Still, if you mean the into to the CONTRIBUTING file, this doc covers a wide range of areas, style guide being only one of them. And an introduction seems to stay a general part for providing info regarding the whole thing, not addressing a specific section. From my pov, style guide info belongs to the style guide chapter, which is actually there, so why not use it. @harshini-rangaswamy , any thought on that?

Reply:
Yeah, so I meant the introduction to the style guide, not the whole contributing file. 

![image](https://user-images.githubusercontent.com/111294980/214322809-8f5fe174-baf7-4255-82aa-3daa55c19906.png)